manifests/params.pp

```diff
@@ -6,7 +6,11 @@
   $graphite_host = '127.0.0.1'
   $graphite_port = 2003
 
-  if $::is_pe {
+  $graphite_prefix = '' # lint:ignore:empty_string_assignment
+  $graphite_suffix = 'puppet'
+  $graphite_reverse_hostname = true
+
+  if str2bool($::is_pe) { # Facts are ALWAYS strings !!
     $config_file  = '/etc/puppetlabs/puppet/graphite.yaml'
```

Comment:
I can't remember where I read that facts are converted to strings, though indeed in 4x, this may change as facts can also be structured. I am using (open source) 3.7.4 in production, and am very certain that the code is broken because $::is_pe is a string. By using str2bool from stdlib >= 4.0.0, the code will accept both string and boolean, solving the issue for all cases.